js/bitrue.js

```diff
@@ -237,9 +238,21 @@ module.exports = class bitrue extends Exchange {
                     'limit': 'FULL', // we change it from 'ACK' by default to 'FULL' (returns immediately if limit is not hit)
                 },
                 'networks': {
-                    'ERC20': 'ETH',
-                    'TRC20': 'TRX',
-                    'TRON': 'TRX',
+                    'SPL': 'SOL',
+                    'SOLANA': 'SOL',
+                    'DOGECOIN': 'DOGE',
+                    'CARDANO': 'ADA',
+                    'ETH': 'ERC20',
+                    'ETHEREUM': 'ERC20',
+                    'TRX': 'TRC20',
+                    'TRON': 'TRC20',
+                    'BNB': 'BEP2',
+                    'BSC': 'BEP20',
+                },
+                'impliedNetworks': {
```

Comment:
I think we might need to do these like it's done in bybit, namely `networks` and `networksById` keys/objects. I see that you probably copied it from binance, but I think that binance's implementation of them might be a bit complex and might be simplified with just `networks` & `networksById` groups, without need of `impliedNetworks`.

however, @carlosmiei mention your thoughts to confirm/reject my thought.

Reply:
@ttodua @AlgoQ yeah I think we can stick to the `networksById` seems pretty self-explanatory to me